### ISSUE TITLE
Only 'alexa' wakeword is reliably stripped

### ISSUE THREAD

**Issue by padrino-x** [Apollon77/ioBroker.alexa2#1210]
ie. even without a gap/pause between it and the command.
The others (computer, echo, amazon and ziggy) end up in the history.summary data point if the pause is not long enough.

**Comment by Apollon77** [comment]
Please provide debug logs for it ... including adapter start. When I remember correctly then the wakewords are only stripped as confgured ... maybe the command is received by another device?

**Comment by padrino-x** [comment]
No, there is no device nearby, and even when I whisper it happens. :)

Debug logs sent via email. =)
